### ISSUE TITLE
Release cycles not being performed correctly.

### ISSUE THREAD

**Issue by inventionlabsSydney** [FiguredLimited/vue-mc#119]
Hi guys,

I love this project but I cannot help but notice the release cycles / what's on the master branch, what's on the releases page and what's on npmjs are all different/inconsistent.

Is there a reason for this? can I help to fix this up?

Thanks,
Karl.

**Comment by inventionlabsSydney** [comment]
Further to this, NPM's current version (0.5.0) is greatly behind the master branch and npm installing from `npm install --save https://github.com/FiguredLimited/vue-mc.git\#master` does not give a correct installation and becomes unusable.

**Comment by nathandaly** [comment]
Not sure this is active/maintained anymore :/

**Comment by rtheunissen** [comment]
Hey everyone, we've had to focus on a lot of other priorities but we've expanded our maintenance pool now so should see more response going forward. This project needs a lot of attention with many open issues to work through, but we're here and will continue to do what we can.

First things first, the last release with a changelog was `0.4.0` but there is no proper tag release for that. The version currently on NPM is `0.5.0` which was tagged on Nov 15, 2018. There have been 35 commits to master since that tag, so we need to consolidate those into a 0.6.0, and establish some scope for a 1.0.0 release that reworks all the big issues that break BC.

We also need to upgrade some dependencies that have security issues - I think this is a good place to start. My plan for this for the week is:

- [ ] Update the tags and changelog to 0.5.0 so that everything is on the same level (except master).
- [ ] Consolidate the commits since then into a new version, also upgrading dependencies.
- [ ] Prioritize some bigger issues and work them into a milestone for 1.0.0

**Comment by inventionlabsSydney** [comment]
Good morning @rtheunissen, thanks for your input and I really appreciate it.

Is it within the realm of possibility to allow others onto this project not involved in Figured?
If so, maybe people like myself and others can jump onboard and help push this project forward.

Thanks again for your update.

**Comment by rtheunissen** [comment]
Absolutely, we'll have an internal discussion about that and get back to you all. 👍

**Comment by thisninja** [comment]
> Hey everyone, we've had to focus on a lot of other priorities but we've expanded our maintenance pool now so should see more response going forward. This project needs a lot of attention with many open issues to work through, but we're here and will continue to do what we can.
> 
> First things first, the last release with a changelog was `0.4.0` but there is no proper tag release for that. The version currently on NPM is `0.5.0` which was tagged on Nov 15, 2018. There have been 35 commits to master since that tag, so we need to consolidate those into a 0.6.0, and establish some scope for a 1.0.0 release that reworks all the big issues that break BC.
> 
> We also need to upgrade some dependencies that have security issues - I think this is a good place to start. My plan for this for the week is:
> 
> * [ ]  Update the tags and changelog to 0.5.0 so that everything is on the same level (except master).
> * [ ]  Consolidate the commits since then into a new version, also upgrading dependencies.
> * [ ]  Prioritize some bigger issues and work them into a milestone for 1.0.0

Hi @rtheunissen,
Are there any updates on that?

**Comment by rtheunissen** [comment]
@thisninja I will be releasing 1.0.0 tonight that will consist of all the changes on `master` since 0.5.0, including a fix for the Vue version being locked. I'm also updating all the severely outdated package dependencies so this is technically a breaking change (some of the date formatting symbols have changed in validation). 

The milestone and roadmap for the next version will be for 2.0.0 then.

**Comment by rtheunissen** [comment]
Hey everyone, I have released 0.6.0 instead, which includes a lot of changes that does technically break semver. ✋ There are some bigger changes incoming to warrant a 1.0.0 from which point on we'll be following semver properly. 

On that note, would anyone be interested in becoming a maintainer on this package? We're looking for some community support to assist.

**Comment by sifex** [comment]
@rtheunissen yea would be keen

**Comment by inventionlabsSydney** [comment]
@rtheunissen we might be keen as well.

**Comment by rtheunissen** [comment]
@sifex and @inventionlabsSydney thanks for the interest. Will get back to you asap. 👍

**Comment by jjag3r** [comment]
Hi, @rtheunissen 
Are there any updates when will 1.0.0. be released?

**Comment by nathandaly** [comment]
Bring the community in with this. It could be so much more popular. It is a great package and I for one find it really useful. 

Allow PR's in the project that are looked at regularly.
